src/modm/board/srxe/board.xml

```diff
@@ -0,0 +1,15 @@
+<library>
+  <repositories>
+    <repository>
+      <path>../../../../repo.lb</path>
+    </repository>
+  </repositories>
+
+  <options>
+    <option name="modm:target">atmega128rfa1-zu</option>
+    <option name="modm:platform:core:f_cpu">16000000</option>
+  </options>
+  <modules>
+    <module>modm:board:srxe</module>
```

Comment:
One extra question - is the `srxe` short name right? Besides Smart Response XE, there are two more devices having some attention from tinkering community:
- Smart Response PE - unlikely to be supported by modm, because it's based on TI's CC2430 and being much less appealing HW (numerical keyboard, simpler screen)
- Smart Response XE Receiver - very similar architecturally to Smart Response XE, but doesn't have a screen and does have a USB connector

I might add a board for the latter at some point. It could be named `srxe-recv` to complement `srxe`.

Reply:
I think it's fine, especially with the example showing how to use it and both being listing and linked on the homepage.

Reply:
The alternatives are `modm:board:smart-response-xe` and `modm:board:smart-response-xe-receiver`, but meh, I don't think it's a hugely important. We can always rename it later with a deprecation alias as done with the black-pill* boards.